package EigeneExceptions is

   Test : exception;

end EigeneExceptions;
